### DISCUSSION TITLE
Remove `pretty_errors`?

### DISCUSSION THREAD

**Discussion by Erotemic** [Lightning-AI/torchmetrics#2544] [Q&A] [answered]
I just spent a non-trivial amount of time wondering why my error messages were highlighted, which wouldn't be a problem by itself, but apparently `pretty_errors` defaults to a state that removes critical traceback information.

It took me forever to find this because I don't use `pretty_errors`. It just happened to occur as an import-time sideeffect when I imported torchmetrics. This behavior should be an opt-in, and should be outside the scope of torchmetrics itself. I recommend simply removing it.

**Comment by Borda** [comment, marked as answer]
it was already removed in `1.4.1` 🐰
